### DISCUSSION TITLE
simulate Read-Write-Read paddern

### DISCUSSION THREAD

**Discussion by mschubi72** [axboe/fio#1729] [Q&A]
Hello Community,

on storage benchmarks is a read-write-read test common. This means that a block is read, written, read. You can recognise a lot, especially with regard to cache strategies etc. Unfortunately, I have no idea how to map this padding in fio. I would like to use this in HCIBench.

Does anyone have an idea or even a solution?

best regards,
Mike

**Comment by sitsofe** [comment]
Hi @mschubi72,

I think the only way to have the current fio generate such a pattern reliably would be by creating an [iolog](https://fio.readthedocs.io/en/latest/fio_doc.html#trace-file-format) and having fio replay it...
